src/db/tables/subscribers.js

```diff
@@ -337,38 +337,6 @@ async function setWaitlistsJoined (options) {
 }
 /* c8 ignore stop */
 
-/**
- * @param {import('../../app/(nextjs_migration)/(authenticated)/user/breaches/breaches.js').Subscriber} subscriber
- */
-// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
-/* c8 ignore start */
-async function removeSubscriber (subscriber) {
```

Comment:
Unrelated, but seems like unused code? Doing some tech debt cleanup

Reply:
Hm, is it a duplicate of `deleteSubscriber`?

Reply:
I think so, it might be older. I searched for it in the codebase and there's no other file referencing it

Reply:
I'd like to get this into heroku to test it via fxa